CSS/layaout.css

```diff
@@ -24,13 +24,18 @@ orden propiedades
 
     font-family: 'Noto Serif';
     font-size: 36px;
+    font-weight: 700;
+    line-height: 39.6px;
 }
 
 .main-subtitle {
     position: relative;
     top: 12rem;
```

Comment:
avoid adding space to the top, instead try to use margins to push the content to the bottom, ie: marging-bottom